interface/ceed-preconditioning.c

```diff
@@ -1126,13 +1127,12 @@ int CeedOperatorGetOperatorAssemblyData(CeedOperator op, CeedOperatorAssemblyDat
 /**
   @brief Create object holding CeedOperator assembly data.
 
-    The CeedOperatorAssemblyData holds an array with references to every active CeedBasis used in the CeedOperator.
-    An array with references to the corresponding active CeedElemRestrictions is also stored.
-    For each active CeedBasis, the CeedOperatorAssemblyData holds an array of all input and output CeedEvalModes for this CeedBasis.
-    The CeedOperatorAssemblyData holds an array of offsets for indexing into the assembled CeedQFunction arrays to the row representing each
-      CeedEvalMode.
-    The number of input columns across all active bases for the assembled CeedQFunction is also stored.
-    Lastly, the CeedOperatorAssembly data holds assembled matrices representing the full action of the CeedBasis for all CeedEvalModes.
+  The CeedOperatorAssemblyData holds an array with references to every active CeedBasis used in the CeedOperator.
+  An array with references to the corresponding active CeedElemRestrictions is also stored.
+  For each active CeedBasis, the CeedOperatorAssemblyData holds an array of all input and output CeedEvalModes for this CeedBasis.
+  The CeedOperatorAssemblyData holds an array of offsets for indexing into the assembled CeedQFunction arrays to the row representing each
+CeedEvalMode. The number of input columns across all active bases for the assembled CeedQFunction is also stored. Lastly, the CeedOperatorAssembly
+data holds assembled matrices representing the full action of the CeedBasis for all CeedEvalModes.
```

Comment:
Good idea and I've gone through in 988944f to try to break sentences onto their own lines where I could find them. I know there is a `ReflowComments` parameter for `clang-format` to disable wrapping of comments, but unfortunately it would also do it for non-docstrings where this can be quite useful I think. So maybe for now it makes sense to keep doing it manually.